### ISSUE TITLE
Feature Request - HDHomerun Support

### ISSUE THREAD

**Issue by breezytm** [twinther/script.tvguide#12]
Hey mate, 

Great addon. I didn't know about it and I wished I found this months ago. I would like to create a settopbox sort of setup with XBMC and my HDHomerun Prime. It works really nice. The only part that sucks is the manual setup. Do you mind adding support for HDHR devices and XMLTV file. Some of them support upnp with the channels name matching the name of the channels in the xmltv.xml file. 

Example:
<display-name>2 WCBS</display-name>

Right now I am matching the stream to the channel manually. I have a total of 630+ channels. Lord have mercy on my soul :D

**Comment by iainsmacleod** [comment]
I don't think you need this to support the XML file, if you could browse the actual HDHomerun box for the strm files, that would be simpler.
